release/scripts/PowerShell/FhirServerRelease/Private/Set-FhirServerApiUsers.ps1

```diff
@@ -58,7 +58,7 @@ function Set-FhirServerApiUsers {
         $userUpn = Get-UserUpn -UserId $userId -TenantDomain $TenantDomain
 
         # See if the user exists
-        $aadUser = Get-AzureADUser -searchstring $userId
+        $aadUser = Get-AzureADUser -Filter "userPrincipalName eq '$userUpn'"
```

Comment:
Looks like there is some additional character here. Deployment is failing.

Reply:
This is weird